specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/notebook.json

```diff
@@ -0,0 +1,384 @@
+				"x-ms-examples": {
+					"CosmosDBNotebookWorkspaceStart": {
+						"$ref": "./examples/CosmosDBNotebookWorkspaceStart.json"
+					}
+				},
+				"description": "Starts the notebook workspace",
+				"parameters": [{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+					},
+					{
+						"$ref": "#/parameters/NotebookWorkspaceNameParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK"
+					},
+					"202": {
+						"description": "Accepted. The start operation will complete asynchronously."
+					}
+				}
+			}
+		}
+	},
+	"definitions": {
+		"NotebookWorkspaceCreateParameters": {
+			"description": "Parameters to create a notebook workspace resource",
+			"type": "object",
+			"allOf": [{
+				"$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ARMProxyResource"
+			}]
+		},
+		"NotebookWorkspaceListResult": {
+			"description": "A list of notebook workspace resources",
+			"type": "object",
+			"properties": {
+				"value": {
+					"type": "array",
+					"description": "Array of notebook workspace resources",
+					"items": {
+						"$ref": "#/definitions/NotebookWorkspace"
+					}
+				}
+			}
+		},
+		"NotebookWorkspace": {
+			"description": "A notebook workspace resource",
+			"type": "object",
+			"properties": {
+				"properties": {
+					"$ref": "#/definitions/NotebookWorkspaceProperties",
+					"description": "Resource properties.",
+					"x-ms-client-flatten": true
+				}
+			},
+			"allOf": [{
+				"$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ARMProxyResource"
+			}]
+		},
+		"NotebookWorkspaceProperties": {
+			"description": "Properties of a notebook workspace resource.",
+			"type": "object",
+			"properties": {
+				"notebookServerEndpoint": {
+					"description": "Specifies the endpoint of Notebook server.",
+					"type": "string",
+					"readOnly": true
+				},
+				"status": {
+					"description": "Status of the notebook workspace. Possible values are: Creating, Online, Deleting, Failed, Updating.",
+					"type": "string",
+					"readOnly": true
+				}
+			}
+		},
+		"NotebookWorkspaceConnectionInfoResult": {
+			"description": "The connection info for the given notebook workspace",
+			"properties": {
+				"authtoken": {
+					"type": "string",
+					"description": "Specifies auth token used for connecting to Notebook server (uses token-based auth)."
+				},
+				"notebookServerEndpoint": {
+					"type": "string",
+					"description": "Specifies the endpoint of Notebook server."
+				}
+			}
+		}
+	},
+	"parameters": {
+		"NotebookWorkspaceNameParameter": {
+			"name": "notebookWorkspaceName",
```

Comment:
How do we indicate in Swagger that only the "default" notebook workspace name is supported?

Reply:
I originally thought that it had to be parameterized for us to not be flagged for "swagger completeness", but I think I was incorrect. I will remove this parameter, force the path to be "/notebookWorkspaces/default", and update examples accordingly.